spec/forms/state_file/id_grocery_credit_form_spec.rb

```diff
@@ -0,0 +1,166 @@
+require "rails_helper"
+
+RSpec.describe StateFile::IdGroceryCreditForm do
+  describe "#valid?" do
+    context "without answering the top-level question" do
+      let!(:intake) { create :state_file_id_intake, :single_filer_with_json }
+      let(:invalid_params) do
+        {
+          household_has_grocery_credit_ineligible_months: "",
+          primary_has_grocery_credit_ineligible_months: "",
+          primary_months_ineligible_for_grocery_credit: "",
```

Comment:
[dust] do we need to include `primary_has_grocery_credit_ineligible_months` and `primary_months_ineligible_for_grocery_credit` in this test?

Reply:
Technically no - the test still passes without them - but they'll always be part of the page and so I'd like to leave them here to be more reflective of real minimal params for this form.